### DISCUSSION TITLE
tasks and jobs finish with fly and ui, but don't return or display results

### DISCUSSION THREAD

**Discussion by HoffiMuc** [concourse/concourse#5793] [General]
I installed concourse via helm on K3s kubernetes.

now I try to execute a simple task and a job (both only a busybox echo "hello world")

```
$ fly -t "${ENV}" execute -c task_hello_world_secret.yml
uploading bootstrapIaC done
executing build 3 at http://concourse.iac.poi/builds/3
```

and here the command hangs and never comes back until I ctrl-c 

but obviously it seems to have succeeded:

```
$ fly -t ${ENV} builds
id  pipeline/job         build  status     start                     end                       duration  team
3   one-off              n/a    succeeded  2020-06-18@17:45:09+0200  2020-06-18@17:45:15+0200  6s        main
```

same via UI and a job (only a busybox echo "hello world" again)

job starts, checks are ok (yellow) and task appears (and starts counting time)

from there nothing happens forever.

but going back to the job, all of a sudden it is green and finished

clicking back on the job, it is green now, but `loading ...` forever without anything appearing.

any ideas?

**Comment by vito** [comment]
Here's a guess: do you have a reverse-proxy that's buffering the response? Build output is sent to the client as an event stream using [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events), so you may need to configure your routing layer to just directly forward traffic.

**Reply by HoffiMuc** [reply]
well I have a local dnsmasq running that forwards the concourse externalUrl to the ingress controller ... how could that effect nothing but the responses??

**Reply by jamieklassen** [reply]
@HoffiMuc sounds a bit like https://github.com/containous/traefik/issues/2576#issuecomment-524579835... not sure what version of traefik is integrated into your k3s

**Reply by HoffiMuc** [reply]
@jamieklassen yep, that was a superb pointer! thanx

`kubectl -n kube-system edit cm traefik` and setting `compress = false` under `[entryPoints.http]` seems to have solved the problem!
